packages/blockchain-extension/tutorials/new-tutorials/basic-tutorials/a1.md

```diff
@@ -1,4 +1,3 @@
-<!-- Test mdToPdf script -->
 **IBM Blockchain Platform**
 
 <img src="./images/ibp.png" alt="IBM Blockchain Platform"></img>
```

Comment:
On Line 151, we may want to tell the user that they may have to change the dropdown on the right from `Tasks` to `Blockchain` in order to see the output from transactions.